ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController_SwiftUI.swift

```diff
@@ -175,8 +179,19 @@ struct NotificationDemoView: View {
                              actionButtonAction: actionButtonAction,
                              showDefaultDismissActionButton: showDefaultDismissActionButton,
                              messageButtonAction: messageButtonAction,
-                             showFromBottom: showFromBottom,
-                             overrideTokens: $overrideTokens.wrappedValue ? NotificationOverrideTokens() : nil)
+                             showFromBottom: showFromBottom)
+            .backgroundGradient(showBackgroundGradient ? backgroundGradient : nil)
+            .overrideTokens($overrideTokens.wrappedValue ? NotificationOverrideTokens() : nil)
+        }
+    }
+
+    private var backgroundGradient: GradientInfo {
+        // It's a lovely blue-to-pink gradient
+        let colors: [DynamicColor] = [DynamicColor(light: ColorValue(0xFFF1FA), dark: ColorValue(0x601044)),
```

Comment:
It's harder to change the text color for just if the gradient is on, so any chance we could switch to orange-to-pink or something for SwiftUI, for better contrast on the blue text?

Reply:
Oh, or maybe just switch to grey-scale if the style uses brand colors, and the lovely blue-to-pink otherwise? Also, would it be possible to get an example of one of those with more than 2 colors?

Reply:
I'd actually rather not try to do any "smarts" on our end about text color -- like all customization, this is something that consuming apps are responsible for ensuring is accessible.

Reply:
That's fair. I also just meant in the demo, not as something the control would try to enforce.

Reply:
While I agree with your point, I also think it's important to not "automatically" switch toggles here. I want people testing the app to see exactly what happens if they put a gradient background with the standard "primary" text -- and why they probably shouldn't do so, since we won't automatically change our style in the component itself once they're consuming it.